datadog/resource_datadog_dashboard.go

```diff
@@ -5657,6 +6088,154 @@ func buildDatadogGroupBySort(sort map[string]interface{}) *datadogV1.LogQueryDef
 	return ddSort
 }
 
+func buildTerraformQuery(datadogQueries []datadogV1.FormulaAndFunctionQueryDefinition) []map[string]interface{} {
+	queries := make([]map[string]interface{}, len(datadogQueries))
+	for i, query := range datadogQueries {
+		terraformQuery := map[string]interface{}{}
+		if query.TimeSeriesFormulaAndFunctionEventQueryDefinition != nil {
+			if dataSource, ok := query.TimeSeriesFormulaAndFunctionEventQueryDefinition.GetDataSourceOk(); ok {
```

Comment:
This is a pretty minor nit, feel free to say no to this one, but can we put `query. TimeSeriesFormulaAndFunctionEventQueryDefinition` in a variable to use in all these followup conditionals. The struct name is pretty long, so it'd drastically reduce the line length 🙂

Reply:
Addressed in commit [5c22cd7](https://github.com/DataDog/terraform-provider-datadog/pull/892/commits/5c22cd771533e25ac58a9e87bc9da024672d34f3).